extension/src/js/background/index.js

```diff
@@ -8,21 +8,104 @@ const constraints = {
 var tabID;
 var port;
 
+var play = false;
+var firstTime = true;
+
+var room;
+var rtcConn2 = null;
+var audioElement = document.createElement('audio');
+    audioElement.setAttribute("preload", "auto");
+    audioElement.load;
 chrome.runtime.onConnect.addListener(function (p) {
     port = p;
+    
     p.onMessage.addListener(function (msg) {
         if (msg.type == "init") {
             socket.emit("create room", msg.roomName);
         }
+        
+        if (msg.type == "play") {
+            if(room != msg.msg){
+                room = msg.msg
+                console.log("Active session with ID: " + room + " found!");
+                socket.emit("new peer", room);
+                setSocketListeners(socket);
+                const rtcConn3 = new RTCPeerConnection(servers);
+                rtcConn3.onicecandidate = event => {
+                    if (!event.candidate) {
+                        console.log("No candidate for RTC connection");
+                        return;
+                    }
+                    socket.emit("peer new ice", {
+                        id: socket.id,
+                        room: room,
+                        candidate: event.candidate
+                    });
+                };
+                rtcConn3.onaddstream = event => {
```

Comment:
onAddStream is being deprecated https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/onaddstream

Reply:
We are now using onAddTrack